system/Config/BaseService.php

```diff
@@ -177,6 +185,44 @@ class BaseService
      */
     private static array $serviceNames = [];
 
+    /**
+     * Simple method to get an entry fast.
+     *
+     * @param string $key Identifier of the entry to look for.
+     *
+     * @return mixed Entry.
+     */
+    public static function get(string $key): mixed
```

Comment:
Applicable review until `override()`.

`mixed` should be limited to `object`, as the static array prop `$instances` accepts `object` as value. I don't think we need to accept anything to be set. Accepting `mixed` may lead to future misuses.

Reply:
Changed to `?object`.